examples/tutorials/scipy2008/traits_image.py

```diff
@@ -38,8 +38,8 @@ def __init__(self):
         self.renderer = plot.img_plot(
             "imagedata",
             name="plot1",
-            xbounds=xgrid,
-            ybounds=ygrid,
+            xbounds=(0, 6),
```

Comment:
Why are the bounds given as `0` and `6` when `x` goes from`0` to `8`?

Reply:
Yeah, it should be the other way around... Just fixed